### ISSUE TITLE
Create list of new and removed questions from former PDF Pension form

### ISSUE THREAD

**Issue by vbellissimo** [department-of-veterans-affairs/va.gov-team#69923]
## Issue Description

As a Pension team,
we are trying to get a sense of what questions on the new online form are new, and what questions were removed, from the former online form,
so that we can get a sense of level of effort for any necessary data mapping required for any Pension API changes.

---
## Tasks

- [x] Look for and document a list of all new questions that were added to the online form
- [x] Look for and document a list of all questions that were removed from the new online form
- [x] Share findings and documentation with Pension engineers and product manager

## Acceptance Criteria
- We will have documented, in a more holistic view, a list that shows us all new questions to the online Pension form we are building and all questions that were removed from the older version of the online Pension form.

**Comment by mayacoforma** [comment]
Hi @vbellissimo 
I have completed the 'diff' audit you asked for, detailing the input fields that have been added or removed on each page. This document is a result of comparing Fiorella's and my files. To enhance accuracy, I recommend scheduling some time for an more comprehensive audit of the paper form, the existing online form, in comparison with the design and the current content in staging.

https://docs.google.com/document/d/1eyL42-PtY6xxWc1-luTwekSNnK6InI_Rs6aN1I7SDVk/edit?usp=sharing
